ksql-rest-app/src/main/java/io/confluent/ksql/rest/server/execution/PullQueryExecutor.java

```diff
@@ -957,4 +967,19 @@ private ColumnReferenceRewriter() {
       return Optional.of(new UnqualifiedColumnReferenceExp(node.getReference()));
     }
   }
+
+  private static final class ConfigRoutingOptions implements RoutingOptions {
+
+    private final KsqlConfig ksqlConfig;
+
+    ConfigRoutingOptions(final KsqlConfig ksqlConfig) {
+      this.ksqlConfig = ksqlConfig;
+    }
+
+    @Override
+    public long getOffsetLagAllowed() {
+      return ksqlConfig.getLong(
+          KsqlConfig.KSQL_QUERY_PULL_STANDBY_READS_MAX_OFFSET_LAG_CONFIG);
```

Comment:
add a note in the configs doc, that this can be overridden per query?

Reply:
Added.